its/plugin/tests/src/test/java/com/sonar/it/php/PhpStanReportTest.java

```diff
@@ -50,6 +50,12 @@ void importReport() {
     assertThat(first.getComponent()).isEqualTo("phpstan_project:src/test.php");
     assertThat(first.getRule()).isEqualTo("external_phpstan:phpstan.finding");
     assertThat(first.getMessage()).isEqualTo("Message for issue without line.");
+    assertThat(first.getImpactsList()).hasSize(1);
+
+    // Not working atm, Impact is not properly send to SQ (somewhere)
```

Comment:
Not sure if I should leave the comments in or just remove them.

Reply:
Please create a verification ticket, attache the discuss thread and just leave a TODO comment with a reference to the ticket.

Reply:
I would suggest to keep tests describing the actual behavior with a remark that it's incorrect with an issue not on our side. If at some point they fix it, we may see these tests failing in the `plugin-DEV` CI workflow